### PR TITLE
About page

### PR DESCRIPTION
**Pull request by x6iae**

### CODE REVIEW

File: src/pages/About/AboutSection.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+
+class AboutSection extends Component {
```
Comment (fiyinadebayo) [inline src/pages/About/AboutSection.jsx]: Not sure if a separate component is needed for this about section, there's really not much styling here. It's just two lines that can well be in the main About component map block.

File: src/services/api-client.js [hunk 2]
```diff
@@ -1,29 +1,26 @@
 import fetchApi from "../utils/fetch-api";
```
Comment (fiyinadebayo) [inline src/services/api-client.js:1]: this update make sense. 👍

File: src/pages/About/AboutSection.jsx (hunk 1 above)
Comment (x6iae) [reply src/pages/About/AboutSection.jsx]: 👍 
Fixed.

Comment (fiyinadebayo) [reply src/pages/About/AboutSection.jsx]: Thanks 👍

**Review by fiyinadebayo — APPROVED** [review]
Great job bro! Thanks.

**Pull request merged**